### DISCUSSION TITLE
articles/2023-04-16-rollup-plugin

### DISCUSSION THREAD

**Discussion by giscus** [olivewind/blog-comments#2] [General]
# articles/2023-04-16-rollup-plugin

作为一名前端开发者，几乎绕不开 Webpack、Rollup、Vite 等打包工具，在遇到复杂场景时我们通常需要编写插件来满足需求。笔者近期刚好遇到一个编写 Rollup 插件的需求，本文是从这次插件开发过程中的要点记录简单整理而来

https://liguo.run/articles/2023-04-16-rollup-plugin

**Comment by olivewind** [comment]
comment test
